target/linux/ramips/dts/mt7621_ubnt_edgerouter-x.dtsi

```diff
@@ -29,33 +29,33 @@
 	ports {
 		port@0 {
 			status = "okay";
-			label = "eth0";
+			label = "wan";
 		};
 
 		port@1 {
 			status = "okay";
-			label = "eth1";
+			label = "lan1";
```

Comment:
Why not use uppercase, EthN or ETHN. I guess the label is handled by C language.

Reply:
This would work, but looks a bit odd especially when wan ends up eth0 (but Eth0 in web ui) and lan ports as EthN. could also be a bit error prone if people try to make rules for eth1 instead of Eth1?

```
3: eth0: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN qlen 1000
    link/ether ac:8b:a9:a4:56:a0 brd ff:ff:ff:ff:ff:ff
4: Eth1@dsa: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue master br-lan state LOWERLAYERDOWN qlen 1000
    link/ether ac:8b:a9:a4:56:a1 brd ff:ff:ff:ff:ff:ff
5: Eth2@dsa: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master br-lan state UP qlen 1000
    link/ether ac:8b:a9:a4:56:a2 brd ff:ff:ff:ff:ff:ff
6: Eth3@dsa: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue master br-lan state LOWERLAYERDOWN qlen 1000
    link/ether ac:8b:a9:a4:56:a3 brd ff:ff:ff:ff:ff:ff
7: Eth4@dsa: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue master br-lan state LOWERLAYERDOWN qlen 1000
    link/ether ac:8b:a9:a4:56:a4 brd ff:ff:ff:ff:ff:ff
 ```